en/docs/manage-apis/deploy-and-publish/deploy-on-gateway/federated-gateways/deploy-on-aws-api-gateway.md

```diff
@@ -95,29 +114,32 @@ AWS APIs are secured using AWS OAuth2 policy. The policy takes in a `Lambda ARN`
         }
         ```
 
-!!!note
-    Below steps would require additional permissions to be granted to the IAM user created in Step 1. Attach `AWSLambda_FullAccess` permission to the user and attach below custom policy to allow sts:AssumeRole action.
-    ```
-    {
-        "Version": "2012-10-17",
-        "Statement": [
-            {
-            "Sid": "test",
-            "Effect": "Allow",
-            "Action": [
-               "iam:GetRole",
-               "iam:PassRole"
-            ],
-            "Resource": "*"
-            }
-        ]
-    }
-    ```
+4. In WSO2 API Publisher portal, navigate to the `Policies` section of the API created in Step 4 and move to the API level policies tab.
+
+5. Attach the AWS OAuth2.0 policy to the API. Here you will have to provide the ARN of the Lambda function and the execution role created above.
 
-1. Navigate to the `Policies` section and move to the API level policies tab.
-2. Attach the AWS OAuth2.0 policy to the API. Here you will have to provide an OAuth2.0 Lambda function ARN and a valid Invoke Role.
     [![attach aws oauth policy]({{base_path}}/assets/img/deploy/attach-aws-oauth-policy.png){: style="width:70%"}]({{base_path}}/assets/img/deploy/attach-aws-oauth-policy.png)
-3. Save and re-deploy the API.
+
+## Step 6 : Deploy and Publish API
+
+1. Navigate to `Deployments` and deploy the API to the AWS API Gateway configured in Step 2.
+
+2. Publish the API to developer portal. 
+
+## Step 7 : Invoke the API
+
+1. Sign in to the Developer Portal.
+
+    `https://<hostname>:9443/developer`
```

Comment:
Shall we change the url to https://<hostname>:9443/devportal?

Reply:
fixed with https://github.com/wso2/docs-apim/pull/9205/commits/00f3cc35ed707d8ac8ae03fd09f65e7091116e12